### PR TITLE
Update ko.json

### PR DESCRIPTION
**Pull request by hoehoetvhss**

### CODE REVIEW

**Comment by hoehoetvhss** [comment]
Currently, 40% of the translation has been completed.

**Comment by hoehoetvhss** [comment]
The use of slash commands as commands in Korean differs greatly from existing text commands, so replace them with English.

**Comment by hoehoetvhss** [comment]
So far I've done the job up to line 748 (about 50%).

**Comment by hoehoetvhss** [comment]
I have completed all the Korean translations so far.

The Korean translation modifications will be reflected in the future version.

File: i18n/ko.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
                     "groups": "{{groupOne}} 및 {{groupTwo}}을(를) 현재 {{groups}} 옵션에 추가합니다.",
                     "include": "현재 {{include}} 옵션에 {{groupOne}}을(를) 추가합니다."
                 },
-                "name": "추가"
```
[the remark this answers is not in the document]
Comment (hoehoetvhss) [reply i18n/ko.json:41]: I think running the commands in English is the right direction, even if you choose Korean.

[the remark this answers is not in the document]
Comment (hoehoetvhss) [reply i18n/ko.json:41]: You can change it if you want to run the command in Korean.

[the remark this answers is not in the document]
Comment (taahamahdi) [reply i18n/ko.json:41]: You bring up a good point with using English here, thanks. I'm thinking to spend a bit more time with the Korean slash command names in place, and if it is causing issues, to revert back to English in the future. I can update the PR to use the Korean slash command names

The rest of the PR looks great, thank you so much for the help!

[the remark this answers is not in the document]
Comment (Brainicism) [reply i18n/ko.json:41]: @hoehoetvhss Do you know how other bots usually handle this? Are their localized commands still in english?

[the remark this answers is not in the document]
Comment (hoehoetvhss) [reply i18n/ko.json:41]: @Brainicism The bots that support slash commands that I know use a method of changing only the message output after executing the command without changing the command execution method even if localization is performed.

**Comment by taahamahdi** [comment]
@hoehoetvhss are we good to merge this? I merged your other PR with slash command names changed to Korean into this already

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**